src/cluster_slot_stats.h

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Valkey Contributors.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD 3-Clause
+ */
+
+#include "server.h"
+#include "cluster.h"
+
+void clusterSlotStatReset(int slot);
+void clusterSlotStatsReset(void);
```

Comment:
These names are too similar. Can we call the second one `clusterSlotStatResetAll`?